documentation/asciidoc/topics/yaml/prometheus_service_monitor.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
```

Comment:
@rigazilla , @ryanemerson just for idea, can we create ServiceMonitor automatically on behalf of user

Reply:
I think so. The simple solution is to just create the `ServiceMonitor` in the same namespace as the Infinispan CR and it's up to the user to configure their `Prometheus` `serviceMonitorNamespaceSelector` so that the required namespaces are scanned.

Reply:
+1
sounds good to me!

Reply:
@ryanemerson Can I resolve this comment? Seems like something that needs to be coded rather than documented. Should we create an issue for tracking?

Reply:
Sure. I've created #959